RELEASE_NOTES.md

```diff
@@ -3,7 +3,7 @@
 - feat: `continue` statements. Like Dafny's `break` statements, these come in two forms: one that uses a label to name the continue target and one that specifies the continue target by nesting level. See section [19.2](https://dafny-lang.github.io/dafny/DafnyRef/DafnyRef#sec-break-continue) of the Reference Manual. (https://github.com/dafny-lang/dafny/pull/1839)
 - feat: The keyword syntax for functions will change in Dafny version 4. The new command-line option `/functionSyntax` (see `/help`) allows early adoption of the new syntax. (https://github.com/dafny-lang/dafny/pull/1832)
 - fix: No warning "File contains no code" if a file only contains a submodule (https://github.com/dafny-lang/dafny/pull/1840)
-
+- fix: "assertion might not hold" instead of "assertion violation" (https://github.com/dafny-lang/dafny/pull/1862)
```

Comment:
```suggestion
- fix: "assertion might not hold" instead of "assertion violation" (https://github.com/dafny-lang/dafny/pull/1862)

```

Reply:
I think we should try to make these release notes self-contained sentences.  (And I think this is an enhancement; are calling these "fixes"?)

Something like *fix: Report unprovable assertions as "assertion might not hold" instead of "assertion violation"*.

Also, we can write #1862 instead of the full URL.

Reply:
I added @fabiomadge 's suggested newline.
I reworded it like `The error "assertion violation" is replaced by the better wording "assertion might not hold". This indicates better that the verifier is unable to prove the assertion.`, is that ok?